### ISSUE TITLE
Autocomplete audit doesn't work when running with headless mode

### ISSUE THREAD

**Issue by sisidovski** [GoogleChrome/lighthouse#11580]
#### Provide the steps to reproduce
1. Run LH on https://en-hyouban.com/user/register/ on CLI with `--chrome-flags="--headless"` option, also add `--preset=experimental` to do the autocomplete audit.
  

#### What is the current behavior?

`<input> elements correctly use autocomplete` is appeared in the "Not applicable" section in best practice.

![Screen Shot 2020-10-20 at 14 19 29](https://user-images.githubusercontent.com/1642255/96543412-49cb4f80-12df-11eb-8eae-b121cc70a007.png)

#### What is the expected behavior?

The audit should fail and show `<input> elements do not have correct autocomplete attributes - 2 elements found` because given page doesn't have autocomplete attributes for each input element. And it works correctly, meaning fails as I expect if I run the command without headless option.

![Screen Shot 2020-10-20 at 14 17 11](https://user-images.githubusercontent.com/1642255/96543342-21435580-12df-11eb-9d04-c42ac95a749a.png)

We'd like to use the autocomplete audit in a cloud function to provide automated UX testing service as a web interface. So headless support is very important to achieve that. Is there any solution to fix that?

#### Environment Information
* Affected Channels: CLI
* Lighthouse version: 6.4.1
* Chrome version: 86.0.4240.80 (Official Build) (x86_64)
* Node.js version: v12.16.1
* Operating System: Mac OS

**Related issues**
https://github.com/GoogleChrome/lighthouse/pull/11342
And also filed a bug to chromium. https://bugs.chromium.org/p/chromium/issues/detail?id=1136798

**Comment by patrickhulce** [comment]
Thanks for filing and for the upstream Chromium bug! :) Our long-term solution here is going to be exposing that same information over the DevTools protocol instead of relying on the attributes, so hopefully that will also address this problem.

> We'd like to use the autocomplete audit in a cloud function to provide automated UX testing service as a web interface. So headless support is very important to achieve that. Is there any solution to fix that?

Can you use `xvfb`? Many server use cases of lighthouse tend to use xvfb instead of headless to avoid additional bot detection strategies that only apply to headless. 

Basically `lighthouse $URL $FLAGS` just becomes `xvfb-run lighthouse $URL $FLAGS`, after `sudo apt-get install -y xvfb`

**Comment by adamraine** [comment]
Autocomplete is only experimental now. We probably won't change that unless CDP support is added.
